.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -154,6 +154,13 @@ jobs:
         if: always()
         run: bash tests/integration/kubernetes/gha-run.sh cleanup-snapshotter
 
+      - name: Take a post-action
+        if: always()
+        run:
+          if [ -f ${HOME}/script/post_action.sh ]; then
+            ${HOME}/script/post_action.sh sev
+          fi
+
```

Comment:
Thanks @AdithyaKrishnan! We should follow the same pattern as the previous cleanup steps, as in call `gha-run.sh` with some command. 🙂 Ditto for `run-k8s-tests-sev-snp`.

Reply:
What's this script about?

Why not following https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/running-scripts-before-or-after-a-job ?

Reply:
@BbolroC opened https://github.com/kata-containers/kata-containers/issues/9934 aiming to create a convention on post-action scripts among the bare-metal machines. While we don't have a consensus,  @AdithyaKrishnan is implementing the same post-action logic as of some 390x jobs (e.g. https://github.com/kata-containers/kata-containers/blob/main/.github/workflows/run-k8s-tests-on-zvsi.yaml#L92) so SNP/SEV jobs aren't blocked.

Reply:
This is process overload, IMHO.

The script is not shared, anyways. Follow the intructions of the link above, ensure the script is executable, and be happy.

Reply:
Please have a look at https://github.com/kata-containers/kata-containers/issues/9934#issuecomment-2258094441 if you follow Fabi's suggestion. Thanks.

Reply:
I would also highly suggest to follow what @BbolroC and @fidencio are proposing, this way the pre/post actions are runner specific and not GHA specific. GHA should be runner agnostic.

Reply:
We got it working by setting the variables you shared. Closing this PR now.